tests/functional/test_contract_event.py

```diff
@@ -277,6 +277,7 @@ def test_contract_two_events_with_same_name(
 def test_contract_decode_logs_no_abi(owner, contract_instance):
```

Comment:
This might not be the best test case to update

Reply:
Yeah, I was just trying to test the `__call__` function worked, I picked a random test